benchmark/_test-double-benchmarker.js

```diff
@@ -7,7 +7,7 @@ if (!['http', 'http2'].includes(myModule)) {
 
 const http = require(myModule);
 
-const duration = process.env.duration || 0;
+const duration = +process.env.duration;
```

Comment:
Is this variable always guaranteed to be defined? Before this would default to zero in such a case, now it'd be set to `NaN`.

Reply:
It is definitely set in https://github.com/nodejs/node/pull/31396/files#diff-39be6178abea38b67efde815bc709950R213

Ideally, we should not use environment variables at all in this case but I did not want to refactor even more.